Formula/postgresql.rb

```diff
@@ -98,6 +98,9 @@ def post_install
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
+      system "brew", "services", "start", "postgresql"
```

Comment:
[http://stackoverflow.com/questions/40040824/how-to-initialize-a-postgresql-database-without-running-the-postgresql-server](url)
This is a possible way to go without starting and stopping the server, but essentially the same stuff happens in the background.